nestkernel/nest_names.h

```diff
@@ -130,6 +131,9 @@ extern const Name events;          //!< Recorder parameter
 extern const Name ex_spikes;       //!< Number of arriving excitatory spikes
 extern const Name exc_conductance; //!< Recorder parameter
 
+extern const Name Ca;
+extern const Name tau_Ca;
+extern const Name beta_Ca;
```

Comment:
The names are ordered alphabetically. Please sort them as you did in the nest_names.cpp file.